src/src/Utils.js

```diff
@@ -7,4 +7,34 @@ export function delay(duration) {
   return new Promise((resolve) => {
     setTimeout(() => resolve(), duration);
   });
+} 
+
+export function composeUrl(...args) {
+  let url = window.apiBaseUrl;
+  (args || []).forEach(x => {
+      if (x[0] === '/') {
+          url += x;
+      } else {
+          url += '/' + x;
+      }
+  });
+  return url;
+}
+
+
+/**
+ * 
+ * @param {string[]} segments 
+ * @returns 
+ */
+export function composeUrl2(segments) {
```

Comment:
Instead of composeUrl and composeUrl2, give them a suffix that tells what they do differently, so they are easier to differentiate